xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamPVRManager.cpp

```diff
@@ -383,9 +387,7 @@ bool CDVDInputStreamPVRManager::CloseAndOpen(const char* strFile)
   return false;
 }
 
-bool CDVDInputStreamPVRManager::SupportsChannelSwitch(void)
+bool CDVDInputStreamPVRManager::IsOtherStreamHack(void)
 {
-  PVR_CLIENT client;
-  return g_PVRClients->GetPlayingClient(client) &&
-         client->HandlesInputStream();
+  return m_isOtherStreamHack;
```

Comment:
writing this I noticed that I set m_isOtherStreamHack at the wrong place. It needs to be set when trying to open otherStream